### PR TITLE
ath79: Add support for Comfast E314N-v2

### PR DESCRIPTION
**Pull request by grosjo**
Taken code from https://patchwork.ozlabs.org/patch/884850/ that was never pushed by the author, and adapted to ath79.

The Comfast E314N-V2 is a 2.4 GHz 2x2 radio with a built-in directional antenna and a second Ethernet port - very similar to the Ubiquiti NanoStation M2. The Ethernet port features a pass-through PoE capability, enabled or disabled with a slide switch. The radio is built using a Qualcomm/Atheros QCA9531 chipset.

Firmware can be flashed on these units by the following method:
1.) Apply power to the unit
2.) Immediately AFTER applying power, hold down the reset button
3.) The WAN, LAN, and wireless lights will flash - wait three seconds (three flashes) and then release the button.
4.) After a second, the lights will flutter quickly and the unit will be visible at 192.168.1.1. A web page will be available to enable quick and simple uploading and flashing of firmware.

During the boot process, these units also look for a tftp server at 192.168.1.10. If one is present, the firmware can be uploaded as a file called firmware-auto.bin

Signed-off-by: Joan Moreau jom@grosjo.net

### CODE REVIEW

**Comment by grosjo** [comment]
@ynezz : Is the format now OK for you ?

**Comment by ynezz** [comment]
> @ynezz : Is the format now OK for you ?

It is much better now. Could you now please take a look at the closed PRs and try to incorporate my comments as well?

**Comment by grosjo** [comment]
Supposedly all is now included

File: target/linux/ath79/base-files/etc/board.d/01_leds [hunk 1]
```diff
@@ -47,6 +47,9 @@ comfast,cf-e120a-v3)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssimediumhigh" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssihigh" "wlan0" "76" "100"
 	;;
+comfast,cf-e314n-v2)
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:white:wan" "eth0"
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:white:lan" "eth1"
```
Comment (ynezz) [inline target/linux/ath79/base-files/etc/board.d/01_leds]: @grosjo Missing `;;` here

**Comment by ynezz** [comment]
@grosjo Thanks! Could you please add information about RAM and FLASH memory sizes to your commit message and upload here bootlog (dmesg) from booted device with this DTS (for reference)?

**Comment by grosjo** [comment]
will do when back home (April)

**Comment by grosjo** [comment]
@ynezz How to refresh the github so I do not need to do it all over again due to master changes ?

**Comment by grosjo** [comment]
@ynezz now github shows 285 commits. This is full crap !

isn't there a better way to submit changes to openwrt git maintainers ?

**Comment by grosjo** [comment]
GITHUB IS CRAP

**Comment by Neustradamus** [comment]
Merged commit:
- https://github.com/openwrt/openwrt/commit/abbbecaa73dcb5a36932a224f74488d0c5e3b3cd

Linked to this PR:
- https://github.com/openwrt/openwrt/pull/1955

**Pull request closed without merging**